quint/src/apalache.ts

```diff
@@ -136,8 +146,9 @@ type RunRequest = { cmd: string; config: string }
 
 type RunResponse =
   | { result: 'failure'; failure: { errorType: string; data: string } }
-  // The success data also includes the parsed module, but we don't need it
-  | { result: 'success' }
+  // The success data also includes the parsed module, either as JSON
+  // representing the Apalache IR, ir as a TLA string (if the `TLA` command is used)
+  | { result: 'success'; success: string }
```

Comment:
Should this be something more meaningful?
```suggestion
  | { result: 'success'; parsed_module: string }
```

Reply:
I see. I didn't realize it was from the proto module. I guess it does make sense to have it more general there. Don't worry about it!